### PR TITLE
mclaren570sgt4

### PR DESCRIPTION
**Pull request by Fuzzwah**
Added the mclaren570sgt4, also sorted cars alphabetical

### CODE REVIEW

File: vehicles.ini [hunk 1]
```diff
@@ -5,19 +5,19 @@ audir18,Audi R18,6,4200
...
 bmwm8gte,BMW M8 GTE,6,6800
-bmwm4gt3,BMW M4 GT3,6,6700
 bmwm4gt4,BMW M4 GT4,5,7100
+bmwm4gt3,BMW M4 GT3,6,6650
 bmwz4gt3,BMW Z4 GT3,6,8400
 cadillacctsvr,Cadillac CTS-V Racecar,6,7400
 c6r,Chevrolet Corvette C6.R GT1,6,5700
 c7vettedp,Chevrolet Corvette C7 Daytona Prototype,6,6800
-c8rvettegte,Chevrolet Corvette C8.R GTE,6,6500
+c8rvettegte,Chevrolet Corvette C8.R,6,6800
```
Comment (samkatakouzinos) [inline vehicles.ini:14]: Alias please revert or provide reason
RPM is ok
```suggestion
c8rvettegte,Chevrolet Corvette C8.R GTE,6,6800
```

File: vehicles.ini [hunk 2]
```diff
@@ -5,19 +5,19 @@ audir18,Audi R18,6,4200
...
 latemodel,Chevrolet Monte Carlo SS,4,6000
 dallaradw12,Dallara DW12,6,11700
 dallaraf3,Dallara F3,6,7000
-dallarair01,Dallara iR-01,7,19500
+dallarap217,Dallara P217,6,8250
+dallarair01,Dallara iR01,7,19000
```
Comment (samkatakouzinos) [inline vehicles.ini:19]: Alias please revert or provide reason
RPM is ok
```suggestion
dallarair01,Dallara iR-01,7,19000
```

File: vehicles.ini [hunk 3]
```diff
@@ -39,16 +39,17 @@ formularenault20,Formula Renault 2.0,7,7200
 formularenault35,Formula Renault 3.5,6,9000
 mx5 mx52016,Global Mazda MX-5 Cup,6,6400
 hpdarx01c,HPD ARX-01c,6,9500
-indypropm18,Indy Pro 2000 PM-18,6,8650
+indypropm18,Indy Pro PM-18,6,8650
```
Comment (samkatakouzinos) [inline vehicles.ini:42]: Alias please revert or provide reason
```suggestion
indypropm18,Indy Pro 2000 PM-18,6,8650
```

File: vehicles.ini [hunk 4]
```diff
@@ -39,16 +39,17 @@ formularenault20,Formula Renault 2.0,7,7200
...
 kiaoptima,Kia Optima,6,7000
-lamborghinievogt3,Lamborghini Huracán GT3 EVO,6,7200
+lamborghinievogt3,Lamborghini Huracan GT3 EVO,6,8000
```
Comment (samkatakouzinos) [inline vehicles.ini:44]: Alias please revert or provide reason
RPM is ok
```suggestion
lamborghinievogt3,Lamborghini Huracán GT3 EVO,6,8000
```

File: vehicles.ini [hunk 5]
```diff
@@ -5,19 +5,19 @@ audir18,Audi R18,6,4200
 audir8gt3,Audi R8 LMS,6,7000
 audirs3lms,Audi RS 3 LMS,6,6600
 bmwm8gte,BMW M8 GTE,6,6800
-bmwm4gt3,BMW M4 GT3,6,6700
 bmwm4gt4,BMW M4 GT4,5,7100
+bmwm4gt3,BMW M4 GT3,6,6650
```
Comment (samkatakouzinos) [inline vehicles.ini:9]: RPM is ok

File: vehicles.ini [hunk 6]
```diff
@@ -5,19 +5,19 @@ audir18,Audi R18,6,4200
...
 cadillacctsvr,Cadillac CTS-V Racecar,6,7400
 c6r,Chevrolet Corvette C6.R GT1,6,5700
 c7vettedp,Chevrolet Corvette C7 Daytona Prototype,6,6800
-c8rvettegte,Chevrolet Corvette C8.R GTE,6,6500
+c8rvettegte,Chevrolet Corvette C8.R,6,6800
 latemodel,Chevrolet Monte Carlo SS,4,6000
 dallaradw12,Dallara DW12,6,11700
 dallaraf3,Dallara F3,6,7000
-dallarair01,Dallara iR-01,7,19500
+dallarap217,Dallara P217,6,8250
```
Comment (samkatakouzinos) [inline vehicles.ini:18]: RPM is ok

File: vehicles.ini [hunk 7]
```diff
@@ -39,16 +39,17 @@ formularenault20,Formula Renault 2.0,7,7200
...
-lamborghinievogt3,Lamborghini Huracán GT3 EVO,6,7200
+lamborghinievogt3,Lamborghini Huracan GT3 EVO,6,8000
 legends ford34c,Legends Ford '34 Coupe,5,10000
 legends ford34c rookie,Legends Ford '34 Coupe - Rookie,5,10000
 lotus49,Lotus 49,5,9000
 lotus79,Lotus 79,5,10600
 protrucks pro2lite,Lucas Oil Off Road Pro 2 Lite,3,6800
 protrucks pro2truck,Lucas Oil Off Road Pro 2 Truck,3,6800
 protrucks pro4truck,Lucas Oil Off Road Pro 4 Truck,6,6800
+mclaren570sgt4,McLaren 570s GT4,7,7125
```
Comment (samkatakouzinos) [inline vehicles.ini:52]: Add vehicle is ok

**Review by samkatakouzinos — CHANGES_REQUESTED** [review]
Please see review entries. 
RPM changes are ok. 
Alias changes I wouldn't accept unless you provide reason to do so.

File: vehicles.ini (hunk 7 above)
Comment (samkatakouzinosracing) [reply vehicles.ini:52]: Vehicle added by #38 and #40

**Comment by samkatakouzinosracing** [comment]
Closing. Intent to add vehicle now done by #38

**Pull request closed without merging**